panic: interface conversion: credentials.AuthInfo is insecure.info, not credentials.TLSInfo

当使用hyperbench测试fabric链时，出现panic
```
panic: interface conversion: credentials.AuthInfo is insecure.info, not credentials.TLSInfo

goroutine 12 [running]:
github.com/hyperledger/fabric-sdk-go/pkg/fab/comm.NewStreamConnection({0x7faf0b89c638, 0xc0003ce960}, {0x7faf0b891780, 0xc0001e19d0}, 0xc00070dd98, {0xc00012a480, 0x15}, {0xc000a63600, 0x5, 0x8})
	github.com/hyperledger/fabric-sdk-go@v1.0.1-0.20210927191040-3e3a3c6aeec9/pkg/fab/comm/streamconnection.go:60 +0x51e
github.com/hyperledger/fabric-sdk-go/pkg/fab/events/deliverclient/connection.New({0x7faf0b89c638, 0xc0003ce960}, {0x7faf0b891780, 0xc0001e19d0}, 0x7faf0b85d268, {0xc00012a480, 0xc000680570}, {0xc000a63600, 0x5, 0x8})
	github.com/hyperledger/fabric-sdk-go@v1.0.1-0.20210927191040-3e3a3c6aeec9/pkg/fab/events/deliverclient/connection/connection.go:66 +0x147
github.com/hyperledger/fabric-sdk-go/pkg/fab/events/deliverclient.glob..func2({0x7faf0b89c638, 0xc0003ce960}, {0x7faf0b891780, 0xc0001e19d0}, {0x7faf0b8864e8, 0xc000680570})
	github.com/hyperledger/fabric-sdk-go@v1.0.1-0.20210927191040-3e3a3c6aeec9/pkg/fab/events/deliverclient/deliverclient.go:52 +0x11d
github.com/hyperledger/fabric-sdk-go/pkg/fab/events/client/dispatcher.(*Dispatcher).HandleConnectEvent(0xc000897ef0, {0x7faf0b69f320, 0xc00062e8b0})
	github.com/hyperledger/fabric-sdk-go@v1.0.1-0.20210927191040-3e3a3c6aeec9/pkg/fab/events/client/dispatcher/dispatcher.go:129 +0x1ae
github.com/hyperledger/fabric-sdk-go/pkg/fab/events/service/dispatcher.(*Dispatcher).Start.func1()
	github.com/hyperledger/fabric-sdk-go@v1.0.1-0.20210927191040-3e3a3c6aeec9/pkg/fab/events/service/dispatcher/dispatcher.go:138 +0x2ba
created by github.com/hyperledger/fabric-sdk-go/pkg/fab/events/service/dispatcher.(*Dispatcher).Start
	github.com/hyperledger/fabric-sdk-go@v1.0.1-0.20210927191040-3e3a3c6aeec9/pkg/fab/events/service/dispatcher/dispatcher.go:122 +0x191
```

--- a/go.mod
+++ b/go.mod
@@ -111,4 +111,7 @@ replace golang.org/x/crypto => golang.org/x/crypto v0.0.0-20210921155107-089bfa5
 
 replace golang.org/x/sys => github.com/golang/sys v0.0.0-20220722155257-8c9f86f7a55f
 
+replace google.golang.org/grpc => google.golang.org/grpc v1.29.1
+
+
 go 1.17

--- a/vm/lua/glua/glua_convert.go
+++ b/vm/lua/glua/glua_convert.go
@@ -213,7 +213,13 @@ func getGoValueReflect(L *lua.LState, v lua.LValue, target reflect.Type, convert
 	}
 
 	switch converted := v.(type) {
-	case lua.LBool, lua.LNumber, *lua.LState, lua.LString:
+	case lua.LString:
+		val := reflect.ValueOf(string(converted))
+		if !val.Type().ConvertibleTo(target) {
+			return reflect.Value{}, invalidTypeErr(v, target)
+		}
+		return val.Convert(target), nil
+	case lua.LBool, lua.LNumber, *lua.LState:
 		return convertToTargetType(converted, target, v)
 	case *lua.LNilType:
 		switch target.Kind() {

